packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -416,6 +418,7 @@ const RealAccountSignup = ({
             return;
         }
         closeRealAccountSignup();
+        if (external_url_params?.should_redirect) redirectToLegacyPlatform();
```

Comment:
why checking should_redirect here? It's anyway done in redirectToLegacyPlatform.

Reply:
done ✅